NOTICE

```diff
@@ -1,4 +1,4 @@
 Flow JVM SDK
-Copyright 2019-2021 Dapper Labs, Inc.
+Copyright 2019-2024 Dapper Labs, Inc.
```

Comment:
```suggestion
```
It's an Apache license and so I don't think we put copyright notices in our code